csm/core/agent/csm_agent.py

```diff
@@ -173,7 +173,7 @@ def init():
 
         # RGW S3 IAM user service
         CsmAgent._configure_rgw_s3_iam_users_service()
-
+        CsmAgent._configure_rgw_s3_bucket_service()
```

Comment:
As per discussion, can you please initialize all the s3 services in single function ?
configure_rgw_s3_services()

Reply:
Done